### PR TITLE
Updated Hackathon, HackathonTeam and HackathonTeamRequest models

### PR DESCRIPTION
**Pull request by mistryharsh28**
To solve #5

### CODE REVIEW

File: project_finder_web_app/models.py [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
-	current_members = models.ManyToManyField(User)
-	hackathon = models.OneToOneField(Hackathon, on_delete=models.CASCADE)
-	vacancies = models.PositiveSmallIntegerField(default=3)
-	closed = models.BooleanField(default=False)
-	cut_off_date = models.DateField(default=date.today)
+    name = models.CharField(max_length=50, unique=True)
+    # leader = models.ForeignKey(User, on_delete=models.CASCADE)
+    # current_members = models.ManyToManyField(User)
+    hackathon = models.OneToOneField(Hackathon, on_delete=models.CASCADE)
+    vacancies = models.PositiveSmallIntegerField(default=3)
```
Comment (shahp98) [inline project_finder_web_app/models.py]: Skills? (for the skill req. thingy?)

File: project_finder_web_app/models.py [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
-	status_choices = (
-		('A' : 'Accepted'),
-		('P' : 'Pending'),
-		('R' : 'Rejected'),
-		)
-	status = models.CharField(max_length=1, choices=status_choices, default='P')
+    # sender = models.ForeignKey(User, on_delete=models.CASCADE)
+    team = models.ForeignKey(HackathonTeam, on_delete=models.CASCADE)
+    message = models.TextField(max_length=500)
+    create_date = models.DateField(default=date.today)
```
Comment (shahp98) [inline project_finder_web_app/models.py]: Missed the skills part.

**Comment by mistryharsh28** [comment]
Done with the changes

File: project_finder_web_app/models.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    def __str__(self):
+        return self.name
+
+
+class HackathonTeam(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    # leader = models.ForeignKey(User, on_delete=models.CASCADE)
+    # current_members = models.ManyToManyField(User)
+    hackathon = models.OneToOneField(Hackathon, on_delete=models.CASCADE)
```
Comment (shahp98) [inline project_finder_web_app/models.py]: Why OneToOne? Would'nt that restrict to only one team per hackathon?

Comment (mistryharsh28) [reply project_finder_web_app/models.py]: I have changed it to a ManyToManyField.

**Comment by akksshah** [comment]
pls review the pr and merge it

File: project_finder_web_app/models.py (hunk 3 above)
Comment (shahp98) [reply project_finder_web_app/models.py]: It needs to be Foreign Key. Please read up on OneToOneField, ForeignKey and ManyToMayField ka implementation.

Comment (mistryharsh28) [reply project_finder_web_app/models.py]: Okay done with it

**Comment by shahp98** [comment]
Changes have been incorporated in #24 .

**Pull request closed without merging**